fenix/app/src/main/res/values/preference_keys.xml

```diff
@@ -198,7 +198,7 @@
 
     <!-- Privacy Settings -->
     <string name="pref_key_open_links_in_a_private_tab" translatable="false">pref_key_open_links_in_a_private_tab</string>
-    <string name="pref_key_open_links_in_external_app" translatable="false">pref_key_open_links_in_external_app</string>
+    <string name="pref_key_open_links_in_external_app_2" translatable="false">pref_key_open_links_in_external_app_2</string>
```

Comment:
does this need the '2' suffix?

Reply:
Yes, that's the main reason.  This is in case that if we want to get the value from the original shared preference again.  But thinking about this, I should try to migrate the value.  I'll update this. Thanks